cli/QuickstepCli.cpp

```diff
@@ -135,6 +140,16 @@ int main(int argc, char* argv[]) {
   gflags::ParseCommandLineFlags(&argc, &argv, true);
   google::InitGoogleLogging(argv[0]);
 
+  // Detect the hardware concurrency level.
+  unsigned int num_hw_threads = std::thread::hardware_concurrency();
+
+  // It is possible that the call above fails and returns 0, in which case
+  // the default should be 1.
+  quickstep::FLAGS_num_workers = (num_hw_threads == 0) ? 1 : num_hw_threads;
+  LOG(INFO) << "Number of threads set to "
```

Comment:
No need to log this here. Just a couple of lines below, we print out the number of worker threads on startup already.